ha/util/message_bus.py

```diff
@@ -239,12 +242,21 @@ def register(message_type: str, partitions: int = 1):
             partitions (int): Number of partition.
         """
         admin = MessageBusAdmin(admin_id=MessageBus.ADMIN_ID)
-        try:
-            if message_type not in admin.list_message_types():
-                admin.register_message_type(message_types=[message_type], partitions=partitions)
-        except Exception as e:
-            if "TOPIC_ALREADY_EXISTS" not in str(e):
-                raise(e)
+        for count in range(MESSAGE_BUS_RETRY_COUNT):
+            try:
+                if message_type not in admin.list_message_types():
+                    admin.register_message_type(message_types=[message_type], partitions=partitions)
+                    break
+            except Exception as e:
+                if "TOPIC_ALREADY_EXISTS" not in str(e):
+                    if count == MESSAGE_BUS_RETRY_COUNT - 1:
+                        Log.error(f"Unable to create message bus topic. Max retry exceeded: {e}")
+                        raise(e)
+                    else:
+                        time.sleep(MESSAGE_BUS_SLEEP_TIME)
```

Comment:
Is sleeping for 2s enough? or we need a bigger time interval between the retries? may be 5 - 10 secs?

Reply:
I think it should be enough. Because from the message bus side, there is already a timeout of 60s. 
Reference: https://github.com/Seagate/cortx-utils/pull/836/files

Please suggest if you still feel it is less. Thanks!

Reply:
Okay